examples/ex_iter.jl

```diff
@@ -11,20 +11,20 @@ opts = Dict(:initstep=>0.1,
             :reltol=>1e-5,
             :abstol=>1e-5)
 # pick your solver
-stepper = [ODE.RKIntegratorAdaptive{:rk45},
```

Comment:
I would write `solver` here, in principle you could add `DenseOutput` (an `AbstractSolver`) to the list and everything would work.